saleor/checkout/steps.py

```diff
@@ -1,13 +1,13 @@
 from __future__ import unicode_literals
-
 from django.core.exceptions import ValidationError
+
```

Comment:
This line is not required here, but it will be nice, to separate `__future__` imports with one blank line, but you've removed it.